Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10446,7 +10446,7 @@ def events_error_handler(response):
         """
         try:
             response = response.json()
-            error = res.reason
+            error = response.reason
```

Comment:
@ShahafBenYakir that would override the response object and we would fail on ```AttributeError``` since ```dict``` class does not have the ```reason``` attribute 

one line above (10448) we override the response object, we want to prevent that.

Reply:
This is outdated, it was written before the changes took place, you can ignore it.